mopub-sdk/mopub-sdk-base/release.gradle

```diff
@@ -0,0 +1,104 @@
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'signing'
+
+repositories {
+    maven { url 'http://nexus.skillz.com/content/repositories/thirdparty/' }
+}
+
+def configureReactNativePom(def pom) {
```

Comment:
This name is a tad misleading.

Reply:
yeah i just copied and pasted this from one of our other ones, ill fix it though